crates/oxc_transformer/src/typescript/mod.rs

```diff
@@ -92,4 +125,19 @@ impl<'a> TypeScript<'a> {
             program.body.push(self.ast.module_declaration(export_decl));
         }
     }
+
+    fn has_value_references(&self, name: &Atom) -> bool {
+        let root_scope_id = self.ctx.scopes().root_scope_id();
+
+        self.ctx
+            .scopes()
+            .get_binding(root_scope_id, name)
+            .map(|symbol_id| {
+                self.ctx
+                    .symbols()
+                    .get_resolved_references(symbol_id)
+                    .any(|x| dbg!(x.is_read()) || dbg!(x.is_write()))
```

Comment:
i can't figure it out the usage of references.
I got `is_read() == true` for `A` 
```TypeScript
import A from "lib";
const x: A = 1
```

Reply:
What should be the expected outcome? You can leave this case for me, tell me which test case we should pass, and we'll figure it out together later.